modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -280,6 +280,205 @@
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="enum">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="extends">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="false">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="for">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="func">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="if">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="in">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="is">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="match">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="namespace">
+			<return type="void" />
+			<description>
+				Reserved for future use. This keyword causes a script error when used.
+			</description>
+		</builtin>
+		<builtin name="not">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="null">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="or">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="pass">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="return">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="self">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="signal">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="static">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="super">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="trait">
+			<return type="void" />
+			<description>
+				Reserved for future use. This keyword causes a script error when used.
+			</description>
+		</builtin>
+		<builtin name="true">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="var">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="void">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="when">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="while">
+			<return type="void" />
+			<description>
+			</description>
+		</builtin>
+		<builtin name="yield">
+			<return type="void" />
+			<description>
+				Reserved for future use. This keyword causes a script error when used.
+			</description>
+		</builtin>
```

Comment:
Is `yield` really reserved for future use? I thought it was reserved to show a proper warning for legacy code, so more like reserved for compatibility.

Reply:
A bit of both, but it's mostly reserved for legacy reasons. There are no plans to bring coroutines with `yield` back in the foreseeable future, but there is a desire for them. Indeed the current description is a bit misleading.

Reply:
That's a bit of my problem with showing those reserved words in general, it might be easy to misunderstand them as "this feature is confirmed and will be implemented soon".